File: pset6/mario/mario.py
from cs50 import get_int


def ask():
    for i in range(height):
        print(" "*((height-i)-1), end="")
        for j in range(i+1):
            print("#", end="")
        print()


while True:
    height = get_int("Height: ")
    if height < 9 and height > 0:
        ask()
        break

File: pset1/Hello/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int h;
    do
    {
        h = get_int("Height: ");
    }

    while (h <= 0 || h >= 9);
    for (int s = 0; s < h; s++)
    {
        for (int n = h - 1; n > s; n--)
        {
            printf(" ");
        }
        for (int a = 0; a <= s; a++)
        {
            printf("#");
        }
        printf("\n");
    }
}
